js/package.json

```diff
@@ -29,6 +29,8 @@
     "build:app": "webpack --config webpack/app",
     "build:lib": "webpack --config webpack/libraries",
     "build:dll": "webpack --config webpack/vendor",
+    "build:markdown": "babel-node ./src/jsonrpc/generator/build-markdown.js",
```

Comment:
Kind of on the same accord - I reckon it would make sense to move the generated md files out from `js/src/jsonrpc/docs` to `js/docs`?